conversion/conversion.go

```diff
@@ -582,7 +582,7 @@ func UpdateDDLForeignKeys(ctx context.Context, adminClient *database.DatabaseAdm
 		return nil
 	}
 	msg := fmt.Sprintf("Updating schema of database %s with foreign key constraints ...", dbURI)
-	p := internal.NewProgress(int64(len(fkStmts)), msg, internal.Verbose())
+	p := internal.NewProgress(int64(len(fkStmts)), msg, internal.Verbose(), true)
 
```

Comment:
I wonder whether it's cleaner to initiate Progress struct as following instead

p := &Progress{total: int64(len(fkStmts)), ...}

Just thinking out loud, don't need to change in this PR.

Reply:
Ideally, variables that don't need to be set outside of the struct should not be exposed outside the package. That prevents misuse from parts of code which must not, and makes it easy when performing refactoring.
The approach of using NewProgress - is more reminiscent of OO programming, but i guess in these scenarios it has its merit